tests/PHPUnit/System/expected/test_OneVisitorTwoVisits__CustomVariables.getReservedCustomVariableKeys.xml

```diff
@@ -1,9 +1,4 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <result>
-	<row>_pks</row>
-	<row>_pkn</row>
-	<row>_pkc</row>
-	<row>_pkp</row>
-	<row>_pk_scount</row>
-	<row>_pk_scat</row>
+	<error message="The method 'getReservedCustomVariableKeys' does not exist or is not available in the module '\Piwik\Plugins\CustomVariables\API'." />
```

Comment:
those API that are tagged `@ignore` actually can't be called via HTTP which is acceptable. ideally we should remove these test cases.